sdk/iothub/azure-iot-deviceprovisioningservice/azure/iot/deviceprovisioningservice/__init__.py

```diff
@@ -0,0 +1,14 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
```

Comment:
Is there an apiview.dev review link available for the package?

Reply:
https://apiview.dev/Assemblies/Review/90240b0f26d946da9472062c5b6e72e3